### ISSUE TITLE
[Question]: Anthropic with Bedrock, "Put words in Claude's mouth"

### ISSUE THREAD

**Issue by austinmw** [run-llama/llama_index#10593]
### Question Validation

- [X] I have searched both the documentation and discord for an answer.

### Question

Anthropic recommends to [Put words in Claude's mouth](https://docs.anthropic.com/claude/docs/put-words-in-claudes-mouth#:~:text=When%20using%20Anthropic's%20API%2C%20you,from%20the%20last%20Assistant%3A%20token.) for a more reliable response. 

What is the best way to do this when using the `Bedrock` class?

--- 

For example, if I do the following:

```python
prompt_str = """
Human: Please generate a question about Paul Graham's life regarding the following topic: {topic}

Assistant: Here is a question about Paul Graham's life regarding the above topic:
"""
prompt_tmpl = PromptTemplate(prompt_str)

llm = Bedrock(
    model="anthropic.claude-v2:1",
    region_name="us-east-1",
    model_kwargs={
        "temperature": 0.5,
        "max_tokens_to_sample": 500,
        "stop_sequences": ["\n\nHuman:"],
    },
)

p = QueryPipeline(
    chain=[prompt_tmpl, llm], verbose=True
)

p.run(topic="college")
```

The prompt will actually be formatted in the following unintended way:
```

Human: 
Human: Please generate a question about Paul Graham's life regarding the following topic: graduation

Assistant: Here is a question about Paul Graham's life regarding the above topic:


Assistant: 
```

**Comment by austinmw** [comment]
This does not work with Bedrock.

**Comment by logan-markewich** [comment]
```
from llama_index.core import ChatPromptTemplate
from llama_index.core.llms import ChatMessage, MessageRole

message_templates = [
    ChatMessage(content="You are an expert system.", role=MessageRole.SYSTEM),
    ChatMessage(
        content="Generate a short story about {topic}",
        role=MessageRole.USER,
    ),
]
chat_template = ChatPromptTemplate(message_templates=message_templates)
```

https://docs.llamaindex.ai/en/stable/module_guides/models/prompts/usage_pattern.html#defining-a-custom-prompt

**Comment by austinmw** [comment]
@logan-markewich There's something wrong here. I've tried the following:

```python
from llama_index.llms import Bedrock
from llama_index import ChatPromptTemplate
from llama_index.llms import ChatMessage, MessageRole
from llama_index.query_pipeline.query import QueryPipeline

message_templates = [
    ChatMessage(
        content="You are an expert system.", 
        role=MessageRole.SYSTEM
    ),
    ChatMessage(
        content="Please generate a question about Paul Graham's life regarding the following topic: {topic}",
        role=MessageRole.USER,
    ),
    ChatMessage(
        content="Here is a question about Paul Graham's life regarding the above topic:",
        role=MessageRole.ASSISTANT,
    ),
]
chat_template = ChatPromptTemplate(message_templates=message_templates)

llm = Bedrock(
    model="anthropic.claude-v2:1",
    region_name="us-east-1",
    model_kwargs={
        "temperature": 0.5,
        "max_tokens_to_sample": 500,
        "stop_sequences": ["\n\nHuman:"],
    },
)

p = QueryPipeline(
    chain=[chat_template, llm], verbose=True
)

response = p.run(topic="college")
print(response)
```

But the prompt ends up being:

```
Human: <system>You are an expert system.</system>

Human: Please generate a question about Paul Graham's life regarding the following topic: college

Assistant: Here is a question about Paul Graham's life regarding the above topic:
```

(notice the incorrectly added `Human: ` prefix to the system message)

**Comment by logan-markewich** [comment]
@austinmw ah right. This is because anthropic/bedrock does not support system prompts

**Comment by logan-markewich** [comment]
If this has changed, it may require a PR

**Comment by austinmw** [comment]
Hmmm. Yeah it should support system prompts:

https://docs.anthropic.com/claude/docs/how-to-use-system-prompts

**Comment by briank-rune** [comment]
@logan-markewich 
[Anthropic does support system prompts](https://docs.anthropic.com/claude/docs/how-to-use-system-prompts), you just need to remove `Human:` and the `<system>` tags. I think you just need to update the string [here](https://github.com/run-llama/llama_index/blob/2b6a46d8498b2259fc84385c1167e0b7b3c0fa87/llama-index-legacy/llama_index/legacy/llms/anthropic_utils.py#L34) to not have those things.

**Comment by austinmw** [comment]
Not sure what's up, but I updated both
- `/home/ubuntu/llama_index/llama-index-integrations/llms/llama-index-llms-anthropic/llama_index/llms/anthropic/utils.py`
- `/home/ubuntu/llama_index/llama-index-legacy/llama_index/legacy/llms/anthropic_utils.py` 

in a `pip install -e` of LlamaIndex, removing those sections and still got the same output. I guess I need to look further into how these packages are installed.

**Comment by austinmw** [comment]
I think this should be fixed with #10603
